### ISSUE TITLE
Issue with UnhandledPromiseRejectionWarning: Unhandled promise rejection

### ISSUE THREAD

**Issue by AlirezaAtGeeks** [giorgio-zamparelli/mongo-in-memory#5]
Could you please update line 160 with 

> collection.insertOne(document, function (error, result) {
> 				if (error) {
> 					reject(error);
> 				} else if (result.n === 0) {
> 					reject(new Error("no document was actually saved in the database"));
> 				} else {
> 					resolve(result.ops[0]);
> 				}
> });

Also when I run mocha in recursive mode, it's complaining about `UnhandledPromiseRejectionWarning: Unhandled promise rejection`. What do you suggest?

**Comment by giorgio-zamparelli** [comment]
@AlirezaAtGeeks could you create a Pull Request for me to approve?
[How to create a Pull Request tutorial](https://yangsu.github.io/pull-request-tutorial)

The unhandled promise rejection I think is something you have to manage in your code either by adding a `.catch` or with
`process.on('unhandledRejection', error => {
    console.log('unhandledRejection', error.message);
});`
[Unhandled Promise Rejections in Node.js](http://thecodebarbarian.com/unhandled-promise-rejections-in-node.js.html)

**Comment by AlirezaAtGeeks** [comment]
It seems I don't have permission to push my changes
If you think that is not necessary, I can ignore it.  However, I wanted to follow the addDocument procedure for handling it.

**Comment by giorgio-zamparelli** [comment]
Yes it's supposed to be that way. Its a Pull Request not a Push changes! :)
Read the [tutorial](https://yangsu.github.io/pull-request-tutorial/) and/or Google it!
